src/lib/y2sap/auto.rb

```diff
@@ -177,7 +177,7 @@ def prepare_hana(prod)
         return :abort
       end
       !prod.key?("sapMDC") || prod["sapMDC"] = "no"
-      File.write(@inst_dir + "/ay_q_masterPwd", prod["sapMasterPW"])
+      File.write(@inst_dir + "/ay_q_masterPwd",  prod["sapMasterPW"])
```

Comment:
NP: double space before `prod[...`. Is that intentional?

Reply:
Too much readability :-)